vagrant/roles/setup.prep/tasks/main.yml

```diff
@@ -8,6 +8,16 @@
     name: epel-release
     state: latest
 
+- name: Install Python2 pip
+  yum:
+    name: python2-pip
+    state: installed
```

Comment:
Why don't we put _state_ as `present` ?

Reply:
yeah, usually `latest`. But of course `installed` should be sufficient.

Reply:
We are already update all the packages earlier in the call. I didn't want to trigger a check for an update again for a package where we don't need the latest installed. So I left it at installed.

Reply:
Ok